### ISSUE TITLE
`withVariable` doesn't work on `object`

### ISSUE THREAD

**Issue by sporto** [ghivert/elm-graphql#2]
Your example shows this:

```
  GraphQl.object
    [ GraphQl.field "user"
       ...
    ]
    |> GraphQl.withVariable "id" "INT!"
```

But this doesn't declare the variables in the query. The only way to declare vars is using `named` e.g.

```
  GraphQl.name "Q"
    [ GraphQl.field "user"
       ...
    ]
    |> GraphQl.withVariable "id" "INT!"
```

This is fine I guess but the example is misleading. Or am I missing something?

**Comment by ghivert** [comment]
Thanks for the report, I'll take a look. I'm working on a new version including mutations. I'll see to fix this in the same time!

**Comment by ghivert** [comment]
Issue fixed in 3.0.0! Thanks again!
